### ISSUE TITLE
Lower FiSH ABI version with newest irssi release

### ISSUE THREAD

**Issue by ABCeedee** [falsovsky/FiSH-irssi#52]
I cloned the latest irssi repo and installed it. When I start irssi, I show "Irssi v1.1-g540639e0" as the version number.

I then cloned the repo and rebuilt FiSH following the instructions in the README. When I type '/load fish' I get the following message:

21:08  Irssi: fish/fish is ABI version 7 but Irssi is version 8, cannot load

I came from version 0.8.19 and FiSH was working flawlessly. This may very well be user error, but I figured I would raise an issue here just in case.

**Comment by ABCeedee** [comment]
Just to note - this occurs on an Arch x86_64 installation, and I built the irssi installation from source. For some reason, removing irssi and trying to install irssi via pacman does not seem to work properly on that machine.

However, on my ARMv8 Arch machine, I was able to install irssi v1.0.1 successfully via pacman, and I was able to build and install FiSH successfully. I think this might just be some weird quirk. I'll try and get the pacman installation working on the x86_64 machine in the meantime.
